doc/source/management/data_at_rest_encryption.rst

```diff
@@ -1,804 +1,11 @@
-
-In case the ``percona-innodb-`` key with the requested ID does not exist in the
-keyring, |Percona Server| will create it with version 1. If a new
-``percona-innodb-`` key cannot be created with the requested ID, the whole
-``CREATE TABLE`` statement fails.
-
-
-
-.. rubric:: FORCE_KEYRING
-
-:Availability: This value is **Experimental** quality
-
-New tables are created encrypted and keyring encryption is enforced.
-
-.. rubric:: ONLINE_TO_KEYRING
-
-:Availability: This value is **Experimental** quality
-
-All tables created or altered without the ``ENCRYPTION=NO`` clause
-are encrypted with the latest version of the default encryption key. If a table
-being altered is already encrypted with the master key, the table is recreated
-encrypted with the latest version of the default encryption key.
-
-.. rubric:: ONLINE_TO_KEYRING_FORCE
-
-:Availability: This value is **Experimental** quality
-
-It is only possible to apply the keyring encryption when creating or altering
-tables.
-
-.. note::
-
-   The ``ALTER TABLE`` statement changes the current encryption mode only if you
-   use the ``ENCRYPTION`` clause.
-
-.. seealso::
-
-   |MariaDB| Documentation: ``innodb_encrypt_tables`` Option
-      https://mariadb.com/kb/en/library/xtradbinnodb-server-system-variables/#innodb_encrypt_tables
-
-      ..variable:: table_encryption_privilege_check
-
-        :cli: table-encryption-privilege-check
-        :implemented: version 8.0.16-7
-        :dyn: Yes
-        :scope: Global
-        :vartype: Boolean
-        :default: OFF
-
-      The variable is used when creating or altering a general tablespace or table with an encryption setting different than the :variable:`default_table_encryption`. The default value is `OFF`.
-
-.. seealso::
-
-  MySQL Documentation: ``table_encryption_privilege_check``
-  https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html
-
-.. variable:: innodb_encryption_threads
-
-   :cli: ``--innodb-encryption-threads``
-   :dyn: Yes
-   :scope: Global
-   :vartype: Numeric
-   :default: 0
-
-:Availability: This feature is of **Experimental** quality.
-
-This variable works in combination with the :variable:`default_table_encryption`
-variable set to ``ONLINE_TO_KEYRING``. This variable configures the number of
-threads for background encryption. For the online encryption to work, this
-variable must contain a value greater than **zero**.
-
-.. variable:: innodb_online_encryption_rotate_key_age
-
-   :cli: ``--innodb-online-encryption-rotate-key-age``
-   :dyn: Yes
-   :scope: Global
-   :vartype: Numeric
-   :default: 1
-
-By using this variable, you can re-encrypt the table encrypted using
-KEYRING. The value of this variable determines how frequently the encrypted
-tables should be encrypted again. If it is set to **1**, the encrypted table is
-re-encrypted on each key rotation. If it is set to **2**, the table is encrypted
-on every other key rotation.
```

Comment:
it can be set to 3 etc. So it should end with and If it is set to **2**, the table is re-encrypted on every other key rotation and so on.

Reply:
OK. Thanks. The Management/DARE doc will only be a redirect to the Security folder. I will delete the info in the management/DARE because Sphinx, which we use to publish, does not work with duplicates. I'll be sure to check if your comments are applied to the topics in the Security directory.